### ISSUE TITLE
EVAC gets cancelled if the mission objective is achieved

### ISSUE THREAD

**Issue by AdityaPrakash-26** [long-war-2/lwotc#1433]
I received a "Hack the Workstation in ADVENT Facility" mission. I sent a high mobility soldier ahead to hack the computer while his comrades held the ground a few blocks back and waited for Firebrand to get to the AO.

As soon as he hacked the computer, the mission timer, as well as the EVAC timer disappeared, and I had to re-request EVAC. I am attaching the save files as well.

[XCOM_Saves.zip](https://github.com/long-war-2/lwotc/files/7172321/XCOM_Saves.zip)

**Comment by wdanny69** [comment]
not sure if you tested or played around with the save. but sometimes throwing evac onto objects or cover etc can stop it for some reason. try to throw it out in an open area

**Comment by AdityaPrakash-26** [comment]
@wdanny69 I did not test it extensively, but this has happend on several missions. I will try to throw it in an open area and get back to you.

**Comment by Tedster59** [comment]
Related to the other various "disappearing evac" issues.  This was fixed by https://github.com/long-war-2/lwotc/commit/b71deccbb429fce9a0b3999faae8b18e62c6427e

TLDR of it was LW's Evac tracking object had an error in the "Is Evac Valid" code causing different logic from the "is location valid" code check when you throw the evac.  LW's check runs when Tile data is updated, such as when environment damage is done, or in this case, the hack is completed and the hack tile is updated.
